compute_halo_properties.py

```diff
@@ -29,7 +29,7 @@
 import halo_properties
```

Comment:
A lot of these are probably my fault, but there are various imports here we don't need any more: sys, traceback, h5py, phdf5, psort, halo_properties.

Reply:
I ran https://github.com/PyCQA/autoflake to remove unused imports